packages/autorag/frontend/src/app/components/AutoragRunsTable/AutoragRunsTable.md

```
@@ -35,6 +35,12 @@ const pageSize = 20;
 
 ## Props
 
+## Columns
+
+Name, Description, **Started** (relative time with tooltip showing full date), Status.
+
+## Props
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)